<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <widget class="QDialogButtonBox" name="settingsButtonBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>545</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="CameraGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>405</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>Фотокамера</string>
   </property>
   <widget class="QPushButton" name="cameraConnectButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Подключиться</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cameraDisconnectButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Отключиться</string>
    </property>
   </widget>
   <widget class="QLabel" name="cameraConnectionStatusLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>246</width>
      <height>56</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="calibrationGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>405</width>
     <height>96</height>
    </rect>
   </property>
   <property name="title">
    <string>Калибровка</string>
   </property>
   <widget class="QLabel" name="calibrationReferencePointsDistanceMkmLabel">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>60</y>
      <width>27</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>мкм</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="calibrationReferencePointsDistanceSpinBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>73</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200000</number>
    </property>
    <property name="value">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QLabel" name="calibrationReferencePointsDistanceLabel">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>30</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Расстояние между реперами</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="angleSettingsGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>406</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Настройки ракурсов</string>
   </property>
   <widget class="QLabel" name="angle1SettingsLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>95</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Ракурс 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="angle2SettingsLabel">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>135</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Ракурс 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="angle3SettingsLabel">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>175</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Ракурс 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="angle4SettingsLabel">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>215</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Ракурс 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="angle5SettingsLabel">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>255</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Ракурс 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="angle6SettingsLabel">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>295</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Ракурс 6</string>
    </property>
   </widget>
   <widget class="QLabel" name="angleSettingsPhotoShootingPosLabel">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>30</y>
      <width>126</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Позиция детали для фотосъёмки&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle1PosXSpinBox">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="anglePosXLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle1PosYSpinBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="anglePosYLabel">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>70</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle2PosXSpinBox">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>140</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle2PosYSpinBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle3PosXSpinBox">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>180</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle3PosYSpinBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle4PosXSpinBox">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>220</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle4PosYSpinBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle5PosXSpinBox">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>260</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle5PosYSpinBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle6PosXSpinBox">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>300</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle6PosYSpinBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle5ExtSpinBox">
    <property name="geometry">
     <rect>
      <x>224</x>
      <y>260</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QComboBox" name="angle4ComboBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>219</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>мкм</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>px</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="toleranceUnitsLabel">
    <property name="geometry">
     <rect>
      <x>333</x>
      <y>70</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Единицы</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="buddy">
     <cstring>toleranceUnitsLabel</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="externalToleranceLabel">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Внешн.</string>
    </property>
   </widget>
   <widget class="QComboBox" name="angle2ComboBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>139</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>мкм</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>px</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="angle4ExtSpinBox">
    <property name="geometry">
     <rect>
      <x>224</x>
      <y>220</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QLabel" name="internalToleranceLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Внутр.</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle1ExtSpinBox">
    <property name="geometry">
     <rect>
      <x>224</x>
      <y>100</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle5IntSpinBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>259</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle2ExtSpinBox">
    <property name="geometry">
     <rect>
      <x>224</x>
      <y>140</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle4IntSpinBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>219</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QComboBox" name="angle5ComboBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>259</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>мкм</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>px</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="angle2IntSpinBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>139</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle3IntSpinBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>179</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle3ExtSpinBox">
    <property name="geometry">
     <rect>
      <x>224</x>
      <y>180</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle1IntSpinBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>99</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QComboBox" name="angle1ComboBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>99</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>мкм</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>px</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="angle3ComboBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>179</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>мкм</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>px</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="angleSettingsTolerancesLabel">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>30</y>
      <width>126</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Допуски&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle6IntSpinBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>299</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle6ExtSpinBox">
    <property name="geometry">
     <rect>
      <x>224</x>
      <y>300</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QComboBox" name="angle6ComboBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>299</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>мкм</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>px</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>calibrationReferencePointsDistanceSpinBox</tabstop>
  <tabstop>cameraConnectButton</tabstop>
  <tabstop>cameraDisconnectButton</tabstop>
  <tabstop>angle1PosXSpinBox</tabstop>
  <tabstop>angle1PosYSpinBox</tabstop>
  <tabstop>angle1ExtSpinBox</tabstop>
  <tabstop>angle1IntSpinBox</tabstop>
  <tabstop>angle1ComboBox</tabstop>
  <tabstop>angle2PosXSpinBox</tabstop>
  <tabstop>angle2PosYSpinBox</tabstop>
  <tabstop>angle2ExtSpinBox</tabstop>
  <tabstop>angle2IntSpinBox</tabstop>
  <tabstop>angle2ComboBox</tabstop>
  <tabstop>angle3PosXSpinBox</tabstop>
  <tabstop>angle3PosYSpinBox</tabstop>
  <tabstop>angle3ExtSpinBox</tabstop>
  <tabstop>angle3IntSpinBox</tabstop>
  <tabstop>angle3ComboBox</tabstop>
  <tabstop>angle4PosXSpinBox</tabstop>
  <tabstop>angle4PosYSpinBox</tabstop>
  <tabstop>angle4ExtSpinBox</tabstop>
  <tabstop>angle4IntSpinBox</tabstop>
  <tabstop>angle4ComboBox</tabstop>
  <tabstop>angle5PosXSpinBox</tabstop>
  <tabstop>angle5PosYSpinBox</tabstop>
  <tabstop>angle5ExtSpinBox</tabstop>
  <tabstop>angle5IntSpinBox</tabstop>
  <tabstop>angle5ComboBox</tabstop>
  <tabstop>angle6PosXSpinBox</tabstop>
  <tabstop>angle6PosYSpinBox</tabstop>
  <tabstop>angle6ExtSpinBox</tabstop>
  <tabstop>angle6IntSpinBox</tabstop>
  <tabstop>angle6ComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
